homeassistant/components/zwave_js/device_action.py

```diff
@@ -141,7 +145,9 @@
 )
 
 
-async def async_get_actions(hass: HomeAssistant, device_id: str) -> list[dict]:
+async def async_get_actions(
+    hass: HomeAssistant, device_id: str
+) -> GetAutomationsResult:
```

Comment:
I would like to revert this, for reasoning and the review see: <https://github.com/home-assistant/core/pull/72163#issuecomment-1133865649>

Reply:
I have reworked this to remove the incorrect GetAutomation*** aliases